src/crm/service/opportunities-service.js

```diff
@@ -0,0 +1,49 @@
+import { HttpClient, json } from 'aurelia-fetch-client';
+import { inject } from 'aurelia-framework';
+
+@inject(HttpClient)
+export class OpportunitiesService {
+  constructor() {
+    this.client = new HttpClient;
```

Comment:
you are injecting, but creating a new instance? I suggest sticking with DI approach only